Update Issues template (#63)
KuriusMTL/HackItForward: .github/ISSUE_TEMPLATE/bug_report.md

---

> Please read the following before submitting:

Reviewer: ```suggestion
Please do the following:
```

> - - Please read our Code of Conduct: https://github.com/KuriusMTL/HackItForward/blob/master/CODE_OF_CONDUCT.md

Reviewer: ```suggestion
- Please read our Code of Conduct: https://github.com/KuriusMTL/HackItForward/blob/master/CODE_OF_CONDUCT.md
```

- Please do NOT submit bug reports for questions.
- Please search existing issues to avoid creating duplicates.

> - Detail the steps you took to encounter the bug
> - Explain what the bug consists of

Reviewer: ```suggestion
- Provide the steps you took to encounter the bug.
- Explain what the bug consists of (what happened vs. what you think should have happened).
```
